### PR TITLE
Add mentor bio for rrampage

### PR DESCRIPTION
**Pull request by rrampage**

### CODE REVIEW

File: mentors/python/r.json [hunk 1]
```diff
@@ -14,5 +14,13 @@
...
-  }
+  },
+  {
+  "github_username": "rrampage",
+  "name": "Raunak Ramakrishnan",
+  "link_text": null,
+  "link_url": "https://dev.to/rrampage/",
+  "avatar_url": null,
+  "bio": "I use Python as a power-tool for quick prototyping and for data analysis."
+}
```
Comment (rpottsoh) [inline mentors/python/r.json]:
```suggestion
  }
```

File: mentors/python/r.json [hunk 2]
```diff
@@ -14,5 +14,13 @@
...
     "bio": "I use Python to do data analysis, engineering, physics, and design calculations for work.  I also use Python because it’s simple and clean and nice and I love it!"
-  }
+  },
+  {
+  "github_username": "rrampage",
+  "name": "Raunak Ramakrishnan",
+  "link_text": null,
+  "link_url": "https://dev.to/rrampage/",
+  "avatar_url": null,
+  "bio": "I use Python as a power-tool for quick prototyping and for data analysis."
```
Comment (rpottsoh) [inline mentors/python/r.json]:
```suggestion
    "bio": "I use Python as a power-tool for quick prototyping and for data analysis."
```

File: mentors/python/r.json [hunk 3]
```diff
@@ -14,5 +14,13 @@
...
     "avatar_url": null,
     "bio": "I use Python to do data analysis, engineering, physics, and design calculations for work.  I also use Python because it’s simple and clean and nice and I love it!"
-  }
+  },
+  {
+  "github_username": "rrampage",
+  "name": "Raunak Ramakrishnan",
+  "link_text": null,
+  "link_url": "https://dev.to/rrampage/",
+  "avatar_url": null,
```
Comment (rpottsoh) [inline mentors/python/r.json]:
```suggestion
    "avatar_url": null,
```

File: mentors/python/r.json [hunk 4]
```diff
@@ -14,5 +14,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I use Python to do data analysis, engineering, physics, and design calculations for work.  I also use Python because it’s simple and clean and nice and I love it!"
-  }
+  },
+  {
+  "github_username": "rrampage",
+  "name": "Raunak Ramakrishnan",
+  "link_text": null,
+  "link_url": "https://dev.to/rrampage/",
```
Comment (rpottsoh) [inline mentors/python/r.json]:
```suggestion
    "link_url": "https://dev.to/rrampage/",
```

File: mentors/python/r.json [hunk 5]
```diff
@@ -14,5 +14,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I use Python to do data analysis, engineering, physics, and design calculations for work.  I also use Python because it’s simple and clean and nice and I love it!"
-  }
+  },
+  {
+  "github_username": "rrampage",
+  "name": "Raunak Ramakrishnan",
+  "link_text": null,
```
Comment (rpottsoh) [inline mentors/python/r.json]:
```suggestion
    "link_text": null,
```

File: mentors/python/r.json [hunk 6]
```diff
@@ -14,5 +14,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I use Python to do data analysis, engineering, physics, and design calculations for work.  I also use Python because it’s simple and clean and nice and I love it!"
-  }
+  },
+  {
+  "github_username": "rrampage",
+  "name": "Raunak Ramakrishnan",
```
Comment (rpottsoh) [inline mentors/python/r.json]:
```suggestion
    "name": "Raunak Ramakrishnan",
```

File: mentors/python/r.json [hunk 7]
```diff
@@ -14,5 +14,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I use Python to do data analysis, engineering, physics, and design calculations for work.  I also use Python because it’s simple and clean and nice and I love it!"
-  }
+  },
+  {
+  "github_username": "rrampage",
```
Comment (rpottsoh) [inline mentors/python/r.json]:
```suggestion
    "github_username": "rrampage",
```

**Comment by rrampage** [comment]
@rpottsoh Done :smile:

**Pull request merged**